docs/CHANGELOG.asciidoc

```diff
@@ -60,6 +61,333 @@ Review important information about the {kib} 8.x releases.
+* Adds the single metric viewer embeddable for dashboards ({kibana-pull}175857[#175857]).
+* Adds support for {esql} in Data visualizer ({kibana-pull}174188[#174188]).
+Management::
+* Adds a method of excluding data tiers when getting a field list ({kibana-pull}167946[#167946]).
+Reporting::
+* Adds support for 'auto' value for CSV scrolling duration ({kibana-pull}175005[#175005]).
+* Adds CSV reporting with {esql} in Discover ({kibana-pull}174511[#174511]).
+Observability::
+* Adds actions column ({kibana-pull}175872[#175872]).
+* Adds resource column with tooltip ({kibana-pull}175287[#175287]).
+* Adds support for the Timeslice Metric visualization on the SLO detail page ({kibana-pull}175281[#175281]).
+* Refactor alert table registration and change default columns ({kibana-pull}175119[#175119]).
+* Adds customization for virtual columns and add the 1st virtual column ({kibana-pull}173732[#173732]).
+* Adds a new option, Visualize this query, to the generated {esql} quires in the Elastic Assistant ({kibana-pull}174677[#174677]).
+Security::
+* Adds context information in Risk Summary ({kibana-pull}174574[#174574]).
+* Adds the Asset Criticality selector for new entity flyouts ({kibana-pull}174438[#174438]).
+
+For more information about the features introduced in 8.13.0, refer to <<whats-new,What's new in 8.13>>.
+
+[[enhancements-and-bug-fixes-v8.13.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.13.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.13.0]]
+=== Enhancements
+Alerting::
+* Improve default AlertsTable columns configuration ({kibana-pull}176137[#176137]).
+* Evenly distribute bulk-enabled alerting rules ({kibana-pull}172742[#172742]).
+* Enables the alerts table for cases in the Stack Management ({kibana-pull}172217[#172217]).
+* Implement a mechanism to copy source data into the alerts-as-data documents for ES Query rules ({kibana-pull}171129[#171129]).
+APM::
+* Adds detectors for anomaly rules creation ({kibana-pull}171901[#171901]).
+* Allows Universal Profiling agent to send error frames ({kibana-pull}176537[#176537]).
+* Adds Azure settings ({kibana-pull}176386[#176386]).
+* Adds table search to services, transactions and errors ({kibana-pull}174490[#174490]).
+* Adds memoization to hooks consumed on service inventory page ({kibana-pull}173973[#173973]).
+* Adds stack traces Threads embeddable ({kibana-pull}173905[#173905]).
+* Enhances the diff topN functions ({kibana-pull}173397[#173397]).
+* Updates Indices API to support sourcemap parameters ({kibana-pull}177847[#177847]).
+* Fixes sorting instances table results on server-side ({kibana-pull}174164[#174164]).
+Cases::
+* Adds support default values for required custom fields ({kibana-pull}175961[#175961]).
+* Collects custom fields telemetry metrics for cases plugin in {kib} ({kibana-pull}175425[#175425]).
+* Persists all filter options of the cases table in the URL ({kibana-pull}175237[#175237]).
```

Comment:
Also, this changed. It should be "Persist all filter options of the cases table, including custom fields, in the URL. The filtering is also persisted when navigating back and forth between pages."